extensionsGallery-insider.json

```diff
@@ -1500,14 +1500,14 @@
 					},
 					"versions": [
 						{
-							"version": "0.4.0",
-							"lastUpdated": "6/26/2019",
+							"version": "0.5.0",
+							"lastUpdated": "8/12/2019",
 							"assetUri": "",
 							"fallbackAssetUri": "fallbackAssetUri",
 							"files": [
 								{
 									"assetType": "Microsoft.VisualStudio.Services.VSIXPackage",
-									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/schema-compare/schema-compare-0.4.0.vsix"
+									"source": "https://sqlopsextensions.blob.core.windows.net/extensions/schema-compare/schema-compare-0.5.0.vsix"
```

Comment:
since it depends on new sqltools service should we update Microsoft.AzDataEngine" version dependency too?